superset/examples/helpers.py

```diff
@@ -39,17 +39,25 @@ def get_examples_folder() -> str:
     return os.path.join(app.config["BASE_DIR"], "examples")
 
 
-def update_slice_ids(layout_dict: Dict[Any, Any], slices: List[Slice]) -> None:
-    charts = [
+def update_slice_ids(pos: Dict[Any, Any]) -> List[Slice]:
```

Comment:
Why not return a set if order isn’t relevant?

Reply:
The type wouldn't match with an SQLA n:n relationship property which expects a list.